src/utils/Api.js

```diff
@@ -0,0 +1,76 @@
+export default class Api {
+  constructor({ baseUrl, headers }) {
+    this._baseUrl = baseUrl;
+    this._headers = headers;
+  }
+
+  _checkResponse(res) {
+    return res.ok ? res.json() : Promise.reject(`Error: ${res.status}`);
+  }
+
+  getInitialCards() {
+    return this._request(`${this._baseUrl}/cards`, { headers: this._headers });
+  }
+
+  _request(url, options) {
```

Comment:
Good job

Now you can use `_request` in each method

Reply:
Remaining fetches updated to this._request and extraneous ._checkresponses have been removed.

All functionality remains, no errors